packages/selenium/test/RunPartial/AnalyzeTests.cs

```diff
@@ -81,5 +81,21 @@ public void ShouldReturnCorrectMetadata(string browser)
             Assert.IsNotEmpty(res.TestRunner.Name);
             Assert.IsNotNull(res.ToolOptions);
         }
+
+        [Test, Timeout(120000)]
```

Comment:
Since the tests seem to add 1min 30sec to the test time, we should increase this by double that number just to avoid flakey test runs where the 2min is exceeded. 

```suggestion
        [Test, Timeout(180000)]
```